language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ArrayTypeDescriptorNodeContext.java

```diff
@@ -72,7 +72,7 @@ public ArrayTypeDescriptorNodeContext() {
 
     private Predicate<Symbol> constantFilter() {
         return symbol -> {
-            if (symbol.kind() != SymbolKind.CONSTANT && symbol.kind() != SymbolKind.ENUM_MEMBER) {
+            if (symbol.kind() != SymbolKind.CONSTANT) {
```

Comment:
Btw, why skip enum members for this? 🤔

Reply:
This is for the array length hence avoid the enums